operator: Add a custom metric that collects Lokistacks requiring a schema upgrade (#11513)
grafana/loki: operator/docs/lokistack/sop.md

### Steps

- Increase the number of queriers

## Lokistack Storage Schema Warning

### Impact

> The lokistack has a warning regarding the configured storage schema.

Reviewer: ```suggestion
The LokiStack warns on a newer object storage schema being available for configuration.
```

### Summary

The schema configuration does not contain the most recent schema version and needs an update.

### Severity

`Warning`

### Access Required

- Console access to the cluster

> - Edit access to the deployed operator and Loki namespace:
>   - OpenShift
>     - `openshift-logging` (LokiStack)
>     - `openshift-operators-redhat` (Loki Operator)

Reviewer 2: I think the user only needs access to the LokiStack resource and not the operator?

```suggestion
- Edit access to the namespace where the LokiStack is deployed:
  - OpenShift
    - `openshift-logging` (LokiStack)
```

### Steps

> - Add a new object storage schema V13 in a future EffectiveDate

Reviewer 2: ```suggestion
- Add a new object storage schema V13 with a future EffectiveDate
```
